File: Problems_and_Solutions/day-6/Java/Solution1.java
import java.util.Arrays;
class Solution1 {
    public static void bubbleSort(int arr[]) {
        int n = arr.length;
        for(int i=0; i<n; i++){
            for(int j=0; j<n-i-1; j++){
                if(arr[j] > arr[j+1]){
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }
            }
        }
    }

    public static void main(String[] args){
        int arr[] ={4, 1, 3, 9, 7};
        bubbleSort(arr);
        System.out.println(Arrays.toString(arr));
    }
}

File: Problems_and_Solutions/day-6/cpp/solution 1.cpp
class Solution {
  public:
    // Function to sort the array using bubble sort algorithm.
    void bubbleSort(vector<int>& arr) {
        int temp=0;
        for(int i=0;i<arr.size();i++)
        {
            for(int j=0;j<arr.size()-1;j++)
                if(arr[j]>arr[j+1])
                {
                    temp=arr[j];
                    arr[j]=arr[j+1];
                    arr[j+1]=temp;
                }
        }
    }
};
